### DISCUSSION TITLE
Exercise 1 Release?

### DISCUSSION THREAD

**Discussion by GunnarSKnudsen** [sebastian-hofstaetter/teaching#51] [Advanced IR: Exercise Q&A] [answered]
When will the Exercise 1 be released?

**Comment by sebastian-hofstaetter** [comment]
We are almost ready, we are just sorting out an issue with our server, but it should be resolved soon.

**Comment by sebastian-hofstaetter** [comment, marked as answer]
We resolved the issues and the server is running, but to give everyone a nice weekend, we are going to start the exercise on Monday :)

**Reply by GunnarSKnudsen** [reply]
Thanks for the clarification (and the weekend :))
